src/services/task.list.service.ts

```diff
@@ -57,7 +55,7 @@ export const initTaskList = (companyNumber: string,
 
   return {
     tasks: allTasks,
-    recordDate: toReadableFormat(DateTime.now().toString()),
+    recordDate: csSubmission.data?.confirmationStatementMadeUpToDate,
```

Comment:
i think we'll still need to convert it to readable format here, it should appear on screen like 15 September 2021 rather than 2021-09-15